src/cplscheme/BaseCouplingScheme.hpp

```diff
@@ -381,7 +398,7 @@ class BaseCouplingScheme : public CouplingScheme {
   double _maxTime;
 
   /// current time; _time <= _maxTime
-  double _time = 0;
+  double _time = 0; // @todo remove?
```

Comment:
For what is it needed?

Reply:
This is mainly related to my comment here:

```
double BaseCouplingScheme::getTime() const
{
  // @todo Might be easier to store _timeWindowStartTime and compute _timeWindowStartTime + _computedTimeWindowPart here.
  return _time;
}
```

Goes a bit into the direction of https://github.com/precice/precice/pull/1109, but makes things clearer on the code side.

Reply:
Lets continue this point in #1649. Removed it here.